src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSliderUI.java

```diff
@@ -150,7 +150,9 @@ public class BasicSliderUI extends SliderUI{
 
     /**
      * Constructs a {@code BasicSliderUI}.
+     * @deprecated This constructor will be removed in future release
```

Comment:
This needs a full stop at the end. It's usually added in `@deprecated` tags.
```suggestion
     * @deprecated This constructor will be removed in future release.
```

Should we explain *why* it's deprecated?

Reply:
Second the recommendation for "why".  As a "forRemoval=true" deprecation, it will appear in the "Terminally Deprecated Elements" table on this page: https://download.java.net/java/early_access/jdk24/docs/api/deprecated-list.html.

Reply:
What text would you add ? 
"This constructor was added by accident in JDK 16." ?